### PR TITLE
NPM Time!

### PR DESCRIPTION
**Pull request by ellsclytn**

### CODE REVIEW

File: source/components/Map/index.js [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+      })))
+    ), [])
+
+    this._tourers = tourers.map(
+      (tourer) => this.createTourer(tourer, points)
+    )
+  }
+
+  updateWaypoints (routes = this.props.routes) {
+    console.log('Update Waypoints')
```
Comment (bradparker) [inline source/components/Map/index.js]: Um

Comment (bradparker) [inline source/components/Map/index.js]: Just noticed this

File: source/index.js [hunk 2]
```diff
@@ -1,416 +1,2 @@
...
-  onSelection: () => {},
-  onTourerDeselection: () => {},
-  onWaypointSelection: () => {},
-  tileUrl: 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}.jpg',
-  tileAttribution: '&copy; <a href="http://www.esri.com/">Esri</a>'
-}
-
-export default Map
+export { default as Map } from './components/Map'
+export { default as createConnected } from './container'
```
Comment (bradparker) [inline source/index.js:2]: You'll want to pop:
actions
reducer
constants 

in here also

Comment (ellsclytn) [reply source/index.js:2]: Added.

File: source/index.js [hunk 3]
```diff
@@ -1,416 +1,5 @@
...
+export { default as actions } from './actions'
+export { default as reducer } from './reducer'
+export { default as constants } from './constants'
```
Comment (bradparker) [inline source/index.js]: This may not export anything

File: source/index.js [hunk 4]
```diff
@@ -1,416 +1,5 @@
...
-  onTourerDeselection: () => {},
-  onWaypointSelection: () => {},
-  tileUrl: 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer/tile/{z}/{y}/{x}.jpg',
-  tileAttribution: '&copy; <a href="http://www.esri.com/">Esri</a>'
-}
-
-export default Map
+export { default as Map } from './components/Map'
+export { default as createConnected } from './container'
+export { default as actions } from './actions'
```
Comment (bradparker) [inline source/index.js]: This may not export anything

**Pull request merged**